scrapscript_tests.py

```diff
@@ -4051,5 +4052,51 @@ def test_pretty_print_variant(self) -> None:
         self.assertEqual(pretty(obj), "#x (a -> b)")
 
 
+class ServerCommandTests(unittest.TestCase):
+    def setUp(self) -> None:
```

Comment:
Sorry, another thing I noticed: some tests are hanging because you never shut down the server. You should have whatever the equivalent of setUp is kill the thing

Reply:
This is the only significant blocker; I can't merge until tests pass